samples/vboxwrapper/vbox_common.cpp

```diff
@@ -463,6 +463,36 @@ string VBOX_BASE::read_vm_log(){
     else return "Error in parsing the log file";
 }
 
+void VBOX_BASE::detect_hw_virt() {
+    string command;
+    string output;
+    string vmhw;
+    size_t vmhw_start;
+    size_t vmhw_end;
+
+    command = "list hostinfo";
+    if (vbm_popen(command, output, "VM hw", false, false) == 0) {
+
+        vmhw_start = output.find("Processor supports HW virtualization: ");
+	if (vmhw_start != string::npos) {
+
+            vmhw_start += strlen("Processor supports HW virtualization: ");
+	    vmhw_end = output.find("\n", vmhw_start);
+            vmhw = output.substr(vmhw_start, vmhw_end - vmhw_start);
+	}
+
+	if (vmhw == "no") {
+            vboxlog_msg("HW virtualization not enabled. It is needed to boot the 64-bit image.");
+            boinc_finish(1);
+	}
+	else if (vmhw == "yes")
+            vboxlog_msg("HW virtualization enabled.");
+    }
+    else {
+        vboxlog_msg("ERROR: Cannot read host info.");
+        boinc_finish(1);
+    }
+}
```

Comment:
can you update your code so that it follows the BOINC code style guidelines: https://boinc.berkeley.edu/trac/wiki/CodingStyle

In particular your use of braces and if statements should be made consistent with the code style.  Thanks!